app/src/docs/AccountCard/AccountCardPageStakingSamples.tsx

```diff
@@ -0,0 +1,151 @@
+/* @license Copyright 2023 @paritytech/polkadot-cloud authors & contributors
+SPDX-License-Identifier: GPL-3.0-only */
+
+import {
+  AccountCard,
+  EllipsisProps,
+  ExtraComponentProps,
+  IconProps,
+  TitleProps,
+} from "@packages/cloud-react/lib/recipes/AccountCard";
+import { SimpleEditor } from "../lib/SimpleEditor";
+import { Grid } from "@packages/cloud-react/lib/base/structure/Grid";
+import { HPosition } from "@packages/cloud-react/lib/types";
+
+export const AccountCardPageStakingSamples = () => {
+  const commonCode = `const titleProps: TitleProps = {
+  address: "1f1yYj2bCFhJCTVdeWLDueUsrZynLAaj6jeMy18fjZ7Cr73",
+  name: "The name of the account",
+  justify: "flex-start",
+};
+
+const ellipsisProps: EllipsisProps = {
+  active: true,
+  amount: 7,
+};
+
+const iconProps: IconProps = {
+  size: 20,
+  gridSize: 3,
+  justify: "space-around",
+};
+
+const extraComponentProps: ExtraComponentProps = {
+  component: (
+    <div
+      style={{
+        borderRight: "0.1rem solid white",
+        padding: "0 2rem",
+        color: "white",
+      }}
+    >
+      Nominating
+    </div>
+  ),
+  gridSize: 4,
+  justify: "space-around",
+};`;
+
+  const code = `<Grid row>
+  <Grid column sm={3}></Grid>
+  <Grid column sm={6}>
+    <AccountCard
+      fontSize="x-small"
+      icon={iconProps}
+      ellipsis={ellipsisProps}
+      title={titleProps}
+      extraComponent={extraComponentProps}
+    />
+  </Grid>
+  <Grid column sm={3}></Grid>
+</Grid>`;
+
+  const codeNoCard = `<Grid row>
+  <Grid column sm={3}></Grid>
+  <Grid column sm={6}>
+    <AccountCard
+      noCard // the only diff from the code above
+      fontSize="x-small"
+      icon={iconProps}
+      ellipsis={ellipsisProps}
+      title={titleProps}
+      extraComponent={extraComponentProps}
+    />
+  </Grid>
+  <Grid column sm={3}></Grid>
+</Grid>`;
+
+  const titleProps: TitleProps = {
+    address: "1f1yYj2bCFhJCTVdeWLDueUsrZynLAaj6jeMy18fjZ7Cr73",
+    name: "The name of the account",
+    justify: "flex-start",
+  };
+
+  const ellipsisProps: EllipsisProps = {
+    active: true,
+    amount: 20,
```

Comment:
Could the amount dictate whether it is active? With an undefined `amount` meaning that it is not active.

Reply:
That could be, but it means that if you want to use the default amount, then the prop should be something like:
```tsx
<AccountCard ellipsis={{}} />
```
which makes not much sense;

In addition there can be a use-case that depending on the width ellipsis will be active or not; thus this flag is good to exist